clients/admin-ui/src/features/privacy-requests/configuration/S3StorageConfiguration.tsx

```diff
@@ -24,8 +22,10 @@ interface SecretsStorageData {
 }
 
 const S3StorageConfiguration = ({
-  storageDetails: { auth_method, bucket, format },
-}: any) => {
+  auth_method,
```

Comment:
Not sure what this looks like deconstructed to avoid the lint error in `StorageConfiguration.tsx`